### PR TITLE
III-4500 Document deleting organizer description

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added documentation for deleting the description of an organizer

---
Ticket: https://jira.uitdatabank.be/browse/III-4500

### CODE REVIEW

File: reference/entry.json [hunk 1]
```diff
@@ -3306,6 +3306,36 @@
...
         ]
+      },
+      "delete": {
+        "summary": "",
+        "operationId": "delete-organizer-description",
+        "responses": {
+          "204": {
+            "description": "No Content. The description was deleted."
+          },
+          "401": {
```
Comment (bertramakers) [inline reference/entry.json:3317]: Is a `400` with type `https://api.publiq.be/probs/url/path-parameter-invalid` also possible? If the `language` path parameter is invalid? Or is that not validated?

Comment (bertramakers) [reply reference/entry.json:3317]: The warning we added with this error also raises an interesting question, maybe we copied the approach from offers a bit too much and should not have included the `language` in the URL for these endpoints 🤔 I guess we can leave it like it is for now (but document the error type if relevant), but we should try to find betters ways in the future I think.

Comment (LucWollants) [reply reference/entry.json:3317]: So another approach would be to leave the `language` out of the path and add the language to a JSON body. Would we then support deleting multiple descriptions at once or just one? Same question for updating a description which also works with a `language` inside the path and not inside the body.

Comment (bertramakers) [reply reference/entry.json:3317]: I would leave it like it is for these endpoints but document the possible error response. We'll figure out a better solution next time we run into a case like this 👍

**Review by bertramakers — APPROVED** [review]

**Pull request merged**